From 31c9828ec026e5d9b9122d55bf0aa7cb45bfecc5 Mon Sep 17 00:00:00 2001
From: Terry Jan Reedy <tjreedy@udel.edu>
Date: Sat, 12 Sep 2020 01:51:52 -0400
Subject: [PATCH] bpo-41729: Fix test_winconsole failures (3) and hang
 (GH-22146)

The problems occured with a repository build on machine
with freshly updated Windows 10 Pro.
---
 PC/_testconsole.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/PC/_testconsole.c b/PC/_testconsole.c
index f6fcfcf1964b..b62f21c339aa 100644
--- a/PC/_testconsole.c
+++ b/PC/_testconsole.c
@@ -63,7 +63,7 @@ _testconsole_write_input_impl(PyObject *module, PyObject *file,
     for (DWORD i = 0; i < size; ++i, ++p, ++prec) {
         prec->EventType = KEY_EVENT;
         prec->Event.KeyEvent.bKeyDown = TRUE;
-        prec->Event.KeyEvent.wRepeatCount = 10;
+        prec->Event.KeyEvent.wRepeatCount = 1;
         prec->Event.KeyEvent.uChar.UnicodeChar = *p;
     }
 
